qutebrowser/mainwindow/tabwidget.py

```diff
@@ -652,7 +652,8 @@ def _draw_icon(self, layouts, opt, p):
         icon_state = (QIcon.On if opt.state & QStyle.State_Selected
                       else QIcon.Off)
         icon = opt.icon.pixmap(opt.iconSize, icon_mode, icon_state)
-        p.drawPixmap(layouts.icon.x(), layouts.icon.y(), icon)
+        # 0x0080 = Center Vertically, 0x0004 = Center Horizontally
+        p.drawItemPixmap(layouts.icon, (0x0080 | 0x0004), icon)
```

Comment:
hard coding `0x0080` and `0x0004` is a bad idea (despite the comment). Can you use the constant values defined from qt directly?

The documentation (I think) for the objects you want is here: http://doc.qt.io/qt-5/qt.html#AlignmentFlag-enum

Reply:
Ah, good call ... I misread that configuration. I'll update shortly.